### DISCUSSION TITLE
Sysmon logs are missing event.category and event.dataset

### DISCUSSION THREAD

**Discussion by sleepingbel** [Security-Onion-Solutions/securityonion#8189] [Ideas] [answered]
Hello all,

After installing the new sysmon modular [sysmonconfig.xml](https://github.com/olafhartong/sysmon-modular/blob/master/sysmonconfig.xml), I have seen that multiple sysmon events do not have a event.category and some event.dataset.

This results in a missing value in the default SOC dashboard
![Knipsel](https://user-images.githubusercontent.com/29328171/175811805-c9dde1d2-b715-4da3-8b2e-9b140370c47f.PNG)

After some research I prepose the following event.category and event.dataset values for the sysmon event.code:
Remark only the values in bold where missing

event.dataset | event.code | event.category
-- | -- | --
process_changed_file | 2 | **host, file**
**service_state_change** | 4 | **host**
**process_terminated** | 5 | **host, process**
image_loaded | 7 | **host**
create_remote_thread | 8 | **host**
process_access | 10 | **host, process**
file_create | 11 | **host, file**
registry_create_delete | 12 | **host, registry**
registry_value_set | 13 | **host, registry**
file_create_stream_hash | 15 | **host, file**
config_change | 16 | **host**
**pipe_create** | 17 | **host, pipe**
**pipe_connected** | 18 | **host, pipe**
**process_tampering** | 25 | **host, process**
**file_delete** | 26 | **host, file**
**error_report** | 255 | **host**

Could these be added?

Regards

Bart

**Comment by weslambert** [comment, marked as answer]
An issue has been created to look into updating the fields with the aforementioned data. Thanks!
https://github.com/Security-Onion-Solutions/securityonion/issues/8194

**Comment by 3isenHeiM** [comment]
I also have this issue, thanks for raising it !

**Comment by dougburks** [comment]
This should be resolved in the upcoming Security Onion 2.3.200 release:
https://github.com/Security-Onion-Solutions/securityonion/issues/8194

**Reply by 3isenHeiM** [reply]
Thanks !
